cmd/wallet-server/go.mod

```diff
@@ -11,19 +11,19 @@ require (
 	github.com/coreos/go-oidc v2.2.1+incompatible
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/mux v1.8.0
-	github.com/hyperledger/aries-framework-go v0.1.8-0.20220326012408-071ce8fc905c
-	github.com/hyperledger/aries-framework-go-ext/component/storage/couchdb v0.0.0-20220325184342-8ccd5c996898
-	github.com/hyperledger/aries-framework-go-ext/component/storage/mongodb v0.0.0-20220325184342-8ccd5c996898
-	github.com/hyperledger/aries-framework-go-ext/component/storage/mysql v0.0.0-20220325184342-8ccd5c996898
-	github.com/hyperledger/aries-framework-go-ext/component/vdr/orb v0.1.4-0.20220325184342-8ccd5c996898
+	github.com/hyperledger/aries-framework-go v0.1.8
+	github.com/hyperledger/aries-framework-go-ext/component/storage/couchdb v0.0.0-20220330145438-233eb7999876
+	github.com/hyperledger/aries-framework-go-ext/component/storage/mongodb v0.0.0-20220330145438-233eb7999876
+	github.com/hyperledger/aries-framework-go-ext/component/storage/mysql v0.0.0-20220330145438-233eb7999876
+	github.com/hyperledger/aries-framework-go-ext/component/vdr/orb v0.1.4-0.20220330145438-233eb7999876
```

Comment:
orb should be `v0.1.4-0.20220330151152-6bbd64bde42e`

Reply:
that's complete release version, it should be `233eb799987643ce2226fc20df020a8a6615c8fd`

Correct me if I am wrong

Reply:
for now, lets follow agent-sdk orb version.